packages/ui/utils/errorHandling.ts

```diff
@@ -1,33 +1,48 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { CreateToastFnReturn } from '@chakra-ui/react';
-import LogRocket from 'logrocket';
+import * as Sentry from '@sentry/nextjs';
 
-export const handleGenericError = (e: any, errorToast?: CreateToastFnReturn) => {
-  console.error(e);
-  console.error(JSON.stringify(e));
+export const handleGenericError = ({
+  error,
+  toast,
+  sentryInfo,
+}: {
+  error: any;
+  toast?: CreateToastFnReturn;
+  sentryInfo: { contextName: string; properties: { [key: string]: any } };
```

Comment:
Can we get the actual type from the sentry package somehow instead of defining it ourselves?

Reply:
Seems like they don't have specific type for context

Reply:
![image](https://user-images.githubusercontent.com/45715420/216032507-9de9f72c-2ff5-46a7-a407-75c4b3d2aed4.png)